packages/scoped-elements/src/tag.js

```diff
@@ -0,0 +1,60 @@
+function* counterFunc() {
```

Comment:
Does this really need to be a generator function? It seems a little overkill to increment a counter?

Reply:
Yes, you are right. I'm going to change it